### PR TITLE
Marcos.cifuentes/basic structure

### PR DESCRIPTION
**Pull request by MarcosCGlobant**
I created basic structure for the Calculator Project, Android Studio Project, MVP Structure and basic classes.

### CODE REVIEW

File: app/app/src/main/res/values-w820dp/dimens.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<resources>
+    <!-- Example customization of dimensions originally defined in res/values/dimens.xml
+         (such as screen margins) for screens with more than 820dp of available width. This
+         would include 7" and 10" devices in landscape (~960dp and ~1280dp respectively). -->
+    <dimen name="activity_horizontal_margin">64dp</dimen>
```
Comment (BQuinteros) [inline app/app/src/main/res/values-w820dp/dimens.xml:2]: Pls try to use a utils class and avoid hardcode

File: app/app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<resources>
+    <!-- Default screen margins, per the Android Design guidelines. -->
+    <dimen name="activity_horizontal_margin">16dp</dimen>
```
Comment (BQuinteros) [inline app/app/src/main/res/values/dimens.xml:2]: Pls try to use a utils class and avoid hardcode

File: app/app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    <dimen name="activity_vertical_margin">16dp</dimen>
```
Comment (BQuinteros) [inline app/app/src/main/res/values/dimens.xml:3]: Pls try to use a utils class and avoid hardcode

**Review by BQuinteros — CHANGES_REQUESTED** [review]

File: app/app/src/androidTest/java/com/globant/calculator/android/ExampleInstrumentedTest.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
```
Comment (mauroLonghi) [inline app/app/src/androidTest/java/com/globant/calculator/android/ExampleInstrumentedTest.java]: Pls, remove this unnecessary spaces, thanks.

File: app/app/src/androidTest/java/com/globant/calculator/android/ExampleInstrumentedTest.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+/**
```
Comment (mauroLonghi) [inline app/app/src/androidTest/java/com/globant/calculator/android/ExampleInstrumentedTest.java]: Pls, remove all the auto generated comments in this class, this it is not important.

File: app/app/src/main/res/values-w820dp/dimens.xml (hunk 1 above)
Comment (MarcosCGlobant) [reply app/app/src/main/res/values-w820dp/dimens.xml:2]: hi, it's not hardcode, it's a dimension configuration on a dimen.xml file

File: app/app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (MarcosCGlobant) [reply app/app/src/main/res/values/dimens.xml:2]: hi, it's not hardcode, it's a dimension configuration on a dimen.xml file

File: app/app/src/main/res/values/dimens.xml (hunk 3 above)
Comment (MarcosCGlobant) [reply app/app/src/main/res/values/dimens.xml:3]: hi, it's not hardcode, it's a dimension configuration on a dimen.xml file

File: app/app/src/main/java/com/globant/calculator/android/MainActivity.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+import android.os.Bundle;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.myapplication.R;
+import com.globant.calculator.android.mvp.model.CalculatorModel;
+import com.globant.calculator.android.mvp.presenter.CalculatorPresenter;
+import com.globant.calculator.android.mvp.view.CalculatorView;
+
```
Comment (mauroLonghi) [inline app/app/src/main/java/com/globant/calculator/android/MainActivity.java:9]: Here, the same, please remove this black spaces

File: app/app/src/main/res/values-w820dp/dimens.xml (hunk 1 above)
Comment (BQuinteros) [reply app/app/src/main/res/values-w820dp/dimens.xml:2]: it's okey, my mistake sorry

File: app/app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (BQuinteros) [reply app/app/src/main/res/values/dimens.xml:2]: it's okey, my mistake sorry

File: app/app/src/main/res/values/dimens.xml (hunk 3 above)
Comment (BQuinteros) [reply app/app/src/main/res/values/dimens.xml:3]: it's okey, my mistake sorry

File: app/app/src/main/java/com/globant/calculator/android/MainActivity.java (hunk 6 above)
Comment (MarcosCGlobant) [reply app/app/src/main/java/com/globant/calculator/android/MainActivity.java:9]: Done, thanks

File: app/app/src/androidTest/java/com/globant/calculator/android/ExampleInstrumentedTest.java (hunk 4 above)
Comment (MarcosCGlobant) [reply app/app/src/androidTest/java/com/globant/calculator/android/ExampleInstrumentedTest.java]: Done, thanks

File: app/app/src/androidTest/java/com/globant/calculator/android/ExampleInstrumentedTest.java (hunk 5 above)
Comment (MarcosCGlobant) [reply app/app/src/androidTest/java/com/globant/calculator/android/ExampleInstrumentedTest.java]: Done, thanks

**Review by mauroLonghi — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Pull request merged**